invenio_rdm_migrator/transform/dates.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""JSON field load module."""
+
+from datetime import datetime
+
+
+class DatetimeMixin:
+    """Transform integers to datetime.
+
+    Assumes no timezone (UTC).
+    """
+
+    @staticmethod
+    def _microseconds_to_isodate(data, fields):
+        """Transform int to datetime."""
+        for field in fields:
+            value = data.get(field)
```

Comment:
in some cases the data does not have the key or it is None (e.g. login info). I'm doubting being so permissive. Maybe it's better to check the key exists, but the value is null (then skip); but fail if the key does not exists?

Reply:
I think it's ok in to be permissive here, since e.g. once we have the dict-diffing many keys won't just not have a value, but will also be missing completely.